site/book/04-using-functions/01-declarative-function-execution.md

```diff
@@ -178,5 +178,43 @@ pipeline:
         tier: mysql
 ```
 
+### `selectors`
+
+Selectors can be used to target specific resources for function execution. Selectors 
+follow OR of AND(s) approach where, within each selector, the filters are ANDed 
+and the selected resources are UNIONed with other selected resources. Please go through 
+[resource selector guide] for more information.
+
+Here are the list of available selector properties:
```

Comment:
The book should use an example (i.e. Wordpress) to explain the concept. Walk the reader through the example. Don't start by going into details of the schema. At this point, the reader has no clue what this functionality is, and why they need it.